mage_ai/frontend/oracle/components/Table/index.style.tsx

```diff
@@ -3,28 +3,32 @@ import styled from 'styled-components';
 import light from '@oracle/styles/themes/light';
 import { BORDER_RADIUS } from '@oracle/styles/units/borders';
 import { UNIT } from '@oracle/styles/units/spacing';
+import { ScrollbarStyledCss } from '@oracle/styles/scrollbars';
 import { transition } from '@oracle/styles/mixins';
 
 export const PADDING_SIZE_UNITS = 1.5;
 
 export const TableStyle = styled.div<any>`
+  ${ScrollbarStyledCss}
+
   overflow-y: auto;
   position: relative;
   width: 100%;
   z-index: 3;
-  border-bottom-left-radius: ${BORDER_RADIUS}px;
-  border-bottom-right-radius: ${BORDER_RADIUS}px;
```

Comment:
I figured it was visually more consistent to have all the corners rounded or not instead of just the bottom corners. Let me know if there was a specific reason for just rounding the bottom ones.

`noBorder = false`:
<img width="1014" alt="image" src="https://github.com/mage-ai/mage-ai/assets/8130751/57d19b4d-af52-4cf5-baf2-31ed6317f027">

`noBorder = true`:
<img width="1023" alt="image" src="https://github.com/mage-ai/mage-ai/assets/8130751/a39c52fa-d0a3-475b-abe4-1da7bf873b16">